swaScraper.py

```diff
@@ -159,39 +165,54 @@ def scrape(args):
 
         if args.seniors:
             outbound_fares = browser.find_elements_by_tag_name("ul")[1]
+        elif args.flightnum:
```

Comment:
Don't think this should be an IF-ELSE. If you specify seniors as an argument, you should also be able to specify a flight number.

Reply:
I think what you probably want is something like this:

`        if args.flightnum:
            # Get the li which contains the flight number.
            xpath = '//button[contains(@class,\'flight-number\') and contains(.,"{0}")]/ancestor::*[self::li][1]'.format(
                args.flightnum)
            desiredLI = browser.find_element_by_xpath(xpath)
            outbound_prices = desiredLI.find_elements_by_class_name("currency_dollars")
        else:
            outbound_prices = outbound_fares.find_elements_by_class_name("currency_dollars")`

But that XPath doesn't work for me.

Reply:
Good catch, thanks!